BeVolt/Periphs/src/ADC.c

```diff
@@ -1,57 +1,104 @@
 /** main.c
  * Program for UTSVT BeVolt's Battery Protection System ADC initialization/output
- * @authors Russell Sutton, Garrett Wong, Sijin Woo
+ * @authors Garrett Wong, Sijin Woo, Russell Sutton, Chase Block
  * @lastRevised 10/26/2019
  */
 
 #include <stdint.h>
 #include "stm32f4xx.h"
+#include "stm32f4xx_dma.h"
```

Comment:
You don't have to include stm32f4xx_dma.h. stm32f4xx.h includes everything for you already